lms/djangoapps/teams/tests/test_views.py

```diff
@@ -387,7 +403,7 @@ def test_access(self, user, status):
 
     def test_naming(self):
         new_teams = [
-            self.post_create_team(data=self.build_team_data(name=name))
+            self.post_create_team(data=self.build_team_data(name=name), user=self.create_and_enroll_student())
```

Comment:
Yes, that's why I added this new user generation. The `add_user` method enforces this constraint.